accelerator/core/core.c

```diff
@@ -112,7 +112,7 @@ status_t ta_send_trytes(const iota_config_t* const iconf, const iota_client_serv
   }
 
   // set the value of attach_res->trytes as output trytes result
-  memcpy(trytes, attach_res->trytes, hash_array_len(attach_res->trytes) * sizeof(hash8019_array_p));
+  memcpy(trytes, attach_res->trytes, hash_array_len(attach_res->trytes) * sizeof(flex_trit_t));
```

Comment:
Shouldn't it be `tryte_t` instead of `flex_trit_t`?

Reply:
since the elements in `attach_res->trytes` is actually in type `flex_trit_t`